README.md

```diff
@@ -1,10 +1,76 @@
-# Picover
+# Picover [![Dev branch verification](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/default_branch_verification.yml) [![Pull Request verification](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml/badge.svg)](https://github.com/intive/Picover/actions/workflows/pull_request_verification.yml)
 
 App for documenting memories from Android Guild meetings and parties
 
+## Team:
+
+<p>
```

Comment:
i am wondering, maybe it will be a way to put more images in one row instead of every row for every contributor ?

Reply:
Yes, it is possible I can remove `<p>` and it does look like this: 
![image](https://github.com/intive/Picover/assets/57491794/ae3200ab-1b5b-4aa2-b4ec-3d98ec2d0ee6)
what do you think about it?

Reply:
Ok, I found a better solution for this 
![image](https://github.com/intive/Picover/assets/57491794/7dff9921-f6a7-4962-b349-aab7ed5e6be6)